src/yz_solrq_helper.erl

```diff
@@ -321,30 +304,33 @@ handle_bad_entries(Index, Ops, Entries) ->
 %%      successful ops and applying side-effects to Solr.
 -spec send_solr_single_ops(index_name(), solr_ops()) -> GoodOps :: solr_ops().
 send_solr_single_ops(Index, Ops) ->
-    lists:takewhile(
-      fun(Op) ->
-              try
-                  T1 = os:timestamp(),
-                  ok = yz_solr:index_batch(Index, prepare_ops_for_batch([Op])),
-                  yz_stat:index_end(Index, length(Ops), ?YZ_TIME_ELAPSED(T1)),
-                  true
-              catch _:Err ->
-                      %% TODO This results in double counting index failures when
-                      %% we get a bad request back from Solr.
-                      %% We should probably refine our stats so that
-                      %% they differentiate between bad data and Solr going wonky
-                      yz_stat:index_fail(),
-                      case Err of
-                          {_, Reason, _} when Reason =:= badrequest; Reason =:= bad_data ->
-                              ?ERROR("Updating a single Solr operation failed for index ~p with bad request.", [Index]),
-                              true;
-                          _ ->
-                              ?ERROR("Updating a single Solr operation failed for index ~p with error ~p", [Index, Err]),
-                              yz_fuse:melt(Index),
-                              false
-                      end
-              end
-      end, Ops).
+  lists:takewhile(fun(Op) ->
```

Comment:
It seems like we normally standardize on 4 spaces for indentation in our Erlang code, but indentation here and elsewhere in this PR is only using 2 spaces. I feel like we should probably stick with 4 spaces to keep things consistent.